lib/bundler/cli/pristine.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require "bundler/cli/common"
+
+module Bundler
+  class CLI::Pristine
+    def run
+      ::Bundler.load.specs.each do |spec|
+        gem_name = "#{spec.name} (#{spec.version}#{spec.git_version})"
+
+        if spec.platform != Gem::Platform::RUBY
```

Comment:
Do you mean
`if !spec.platform.nil? && spec.platform != Gem::Platform::RUBY` ?

So that it would only show message for non-ruby platform gems?